scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/examples/gbtClassifierTrainingExampleOnCriteoClickLogsDataset.scala

```diff
@@ -63,7 +63,9 @@ case class Params(
                    simpleAPIKEY: String = "simpleAPIKEY",
                    modelSavePath: String = "/host/data/model",
                    maxIter: Int = 100,
-                   maxDepth: Int = 2
+                   maxDepth: Int = 2,
+                   userName: String = "bkeywhizUserName",
+                   userPassword: String = "bkeywhizUserPassword"
```

Comment:
we should not provide any default value to password
and may rename password to token